demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -175,6 +194,7 @@ def is_file_valid(self):
             bool. True if file's release notes are valid, False otherwise.
         """
         validations = [
+            self.is_breaking_change(),
```

Comment:
let's think of a better name for this method (someone who reads this can think that every release notes file must be with break changes)